game/games/pixel_side_scroller/pawns/pawn.gd

```diff
@@ -4,10 +4,12 @@ var speed: float = 80
 var gravity: float = 9.8
 var jump_force: float = 100.0
 var jump_time: float = 0.2
+var coyote_time: float = 0.15
 var max_fall_speed: float = 2000
 
 var left: float = 0.0
 var right: float = 0.0
+var fall_time: float = 0.0
```

Comment:
I think something like `time_since_jump` makes things more clear. Although a comment would also do the trick.

Reply:
That's actually the time since the player/pawn lost contact with the ground. It's not necessarily a jump.

Reply:
Right, sorry. I would suggest `time_since_ground` / `time_since_on_ground` then. But in that case `fall_time` is also probably fine.

Reply:
Changed it to `air_time`, which should actually describe it way better.